noir-projects/noir-contracts/contracts/token_contract/src/main.nr

```diff
@@ -21,10 +21,12 @@ pub contract Token {
     use dep::aztec::{
         context::{PrivateCallInterface, PrivateContext},
         encrypted_logs::log_assembly_strategies::default_aes128::{
-            event::encode_and_encrypt_event_unconstrained,
+            // TODO(benesjan): uncomment this once the events are updated.
+            // event::encode_and_encrypt_event_unconstrained,
```

Comment:
I have no clue why but event logs suddenly became fed to the process_log function. Didn't bother recalling why this is the case as soon the event logs stuff will use the standard encryption and will be expected to be passed there. So I just disabled event emission and will re-enable it up the stack.

Reply:
>So I just disabled event emission and will re-enable it up the stack.

This will mean we'll break events in master until we fix them though. I think we need to understand what happened to them.

Reply:
What do you mean they became fed to `process_log`? That was happening already, we were simply ignoring those log type ids.

Reply:
I had this idea that before the event logs never managed to travel all the way to the process_log function but with the changes in this PR it started happening and it started erroring out.

But anyway it doesn't matter because in the 3rd PR of the stack I uncomment these emissions.